scripts/load-people

```diff
@@ -150,59 +135,64 @@ sub loadconstituency {
     $twig->purge;
 }
 
-sub loadroyal { load_member(0, @_); }
-sub loadmp { load_member(1, @_); }
-sub loadlord { load_member(2, @_); }
-sub loadni { load_member(3, @_); }
-sub loadmsp {
-    my ($twig, $member) = @_;
-    my $spurl = $member->att('spurl');
-    my $person_id = load_member(4, @_);
-    $dbh->do('replace into personinfo (person_id, data_key, data_value) values (?, ?, ?)', {},
-        $person_id, 'sp_url', $spurl);
+sub load_people_json {
+    my $j = decode_json(read_file($pwmembers . 'people.json'));
+    foreach (@{$j->{organizations}}) {
+        $organizations{$_->{id}} = $_;
+    }
+    foreach (@{$j->{posts}}) {
+        $posts{$_->{id}} = $_;
+    }
+    foreach (@{$j->{memberships}}) {
+        load_member($_);
+    }
 }
 
 my %member_ids = ();
-sub load_member {
-    my ($house, $twig, $member) = @_;
 
-    die "Unknown house" if $house == 1 && $member->att('house') ne "commons";
-    die "Unknown house" if $house == 2 && $member->att('house') ne "lords";
+sub load_member {
+    my ($member) = @_;
 
-    my $id = $member->att('id');
-    my $person_id = $membertoperson{$id};
-    $id =~ s:uk.org.publicwhip/(member|lord|royal)/::;
+    (my $id = $member->{id}) =~ s:uk.org.publicwhip/(member|lord|royal)/::;
+    (my $person_id = $member->{person_id}) =~ s#uk.org.publicwhip/person/##;
 
-    my $fromdate = $member->att('fromdate');
-    $fromdate .= '-00-00' if length($fromdate) == 4;
-    $fromdate = '0000-00-00' unless $fromdate;
-    my $todate = $member->att('todate');
-    $todate .= '-00-00' if length($todate) == 4;
+    my $start_date = $member->{start_date};
+    $start_date .= '-00-00' if length($start_date) == 4;
+    $start_date = '0000-00-00' unless $start_date;
+    my $end_date = $member->{end_date} || '9999-12-31';
```

Comment:
I don't think there actually is anything without a start_date, so it's a bit odd; but yes, might as well :)